charts/resoto/README.md

```diff
@@ -25,7 +25,7 @@ A Helm chart for Kubernetes
 | fullnameOverride | string | `""` |  |
 | image.pullPolicy | string | `"IfNotPresent"` |  |
 | image.repositoryFolder | string | `"somecr.io/someengineering"` |  |
-| image.tag | string | `""` |  |
+| image.tag | string | `"latest"` |  |
```

Comment:
If the default is the `appVersion`, why does this need to be specified? I think it's best to pin to a specific version, in case new releases have breaking changes.

Reply:
This doesn't need to be specified. I have not seen helm-charts hardcoding the tags be it argo, kubecost, prometheus, harbor or take any other CNCF project. Either they specific `latest/stable/dev` or they default to `appVersion` by leaving other empty.